src/app/logout/logout.component.html

```diff
@@ -0,0 +1,27 @@
+<!--
+   Copyright 2020 OICR
+ *
+   Licensed under the Apache License, Version 2.0 (the "License");
+   you may not use this file except in compliance with the License.
+   You may obtain a copy of the License at
+ *
+       http://www.apache.org/licenses/LICENSE-2.0
+ *
+   Unless required by applicable law or agreed to in writing, software
+   distributed under the License is distributed on an "AS IS" BASIS,
+   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+   See the License for the specific language governing permissions and
+   limitations under the License.
+-->
+<app-header>
+  Logged Out
+</app-header>
+<div class="container">
+  <div class="column">
+    <p class="lead">
+      You have logged out of Dockstore. Note that if you are still logged into GitHub or Google, a user may be able to log back into
+      Dockstore without being prompted for credentials. If on a public computer, we recommend also signing out of GitHub and Google. Click
+      <a routerLink="/login">here</a> to login again.
```

Comment:
Probably the log-in again part is unnecessary

Reply:
Re-requesting review just to make sure I understand you. Screenshot at top updated.